src/eslint-plugin-sx/src/rules/__tests__/fixtures/no-unused-stylesheet/invalid/unrelated-call-expression.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @flow
+ * @eslintExpectedError SX function "styles" was not used anywhere in the code.
```

Comment:
Why is the location missing here?

Reply:
It's an opt-in feature and I didn't want to add it everywhere. 🙂